var/spack/repos/builtin/packages/flann/package.py

```diff
@@ -0,0 +1,134 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Flann(CMakePackage):
+    """
+    FLANN is a library for performing fast approximate nearest neighbor
```

Comment:
I will fix it to appease the `spack` gods.  I might look into that, because I wanted to include rst directives in something once for `spack` but the `info` command totally chokes on it.  It doesn't even preserve paragraph endings.  I was going to be doing something over the summer related to running `sphinx` to acquire `manpage` output from rst (e.g. for tables, directives, etc).  If I ever get it I'll ping you back and see if `spack` can benefit.